test/fluree/db/ledger/api/downloaded.clj

```diff
@@ -84,9 +84,8 @@
 
       (is (= 200 status))
 
-      ; The keys in the response are -> :request-time :aleph/keep-alive? :headers :status :connection-time :body
-      (is (not (empty? (remove nil? (map #(#{:request-time :aleph/keep-alive? :headers :status :connection-time :body} %)
-                                         response-keys)))))
+      ; The keys in the response are -> :opts :body :headers :status
+      (is (every? (map #(#{:opts :body :headers :status})) response-keys))
```

Comment:
I think this would be:
(is (every? (map #{:opts :body :headers :status} response-keys)))

Reply:
In this case using a set as a function (http://blog.jayfields.com/2010/08/clojure-using-sets-and-maps-as.html)

Reply:
ah, yep. no need to wrap the set in a fn literal in that context. 👍🏻

Reply:
Yeah, that makes sense. These map key set checks are always slightly less intuitive to me than they seem. And in fact, when I actually threw it a REPL, it was `(every? #{:opts :body :headers :status} response-keys)` w/o the `map`. But what that checks (as you saw) is that every key in the map is one of the keys in the set literal. It doesn't care if one or more of the keys in the set is missing from the map!

Whereas the body of `contains-many?` is just `(every? #(contains? m %) ks)` which is saying "for every key in `ks`, check that the map `m` contains it" and is really what we want here. Good fix!